src/dao/AppDAO.js

```diff
@@ -151,7 +155,19 @@ class AppDAO extends AbstractContractDAO {
     };
 
     getLOCs = (account: string) => {
```

Comment:
Don't rename, just extract. AppDAO now also have DAO factory and member profile logic, and there is a calls to AppDAO in another DAOs. So we need to deal with all of this and then we'll be able to just remove AppDAO at all. Probably this is my issue.